lang/cs/Org.Apache.REEF.Driver/Bridge/Events/FailedTask.cs

```diff
@@ -16,19 +16,26 @@
 // under the License.
 
 using System;
+using System.IO;
 using System.Runtime.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
 using Org.Apache.REEF.Common.Avro;
+using Org.Apache.REEF.Common.Exceptions;
 using Org.Apache.REEF.Driver.Bridge.Clr2java;
 using Org.Apache.REEF.Driver.Context;
 using Org.Apache.REEF.Driver.Task;
 using Org.Apache.REEF.Utilities;
+using Org.Apache.REEF.Utilities.Diagnostics;
 using Org.Apache.REEF.Utilities.Logging;
 
 namespace Org.Apache.REEF.Driver.Bridge.Events
 {
     internal sealed class FailedTask : IFailedTask
     {
-        private static readonly Logger LOGGER = Logger.GetLogger(typeof(FailedTask));
+        private static readonly Logger Logger = Logger.GetLogger(typeof(FailedTask));
+        
+        private readonly BinaryFormatter _formatter = new BinaryFormatter();
```

Comment:
If BinaryFormatter is only used in GetCause(), you might just create and use it in that method.